app/views/answers/_index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% answers.each do |answer| %>
+	<%= answer.content %><br>
+	<%= answer.total_answer_votes %> Votes
+  <p><b>Answer Comments:</b></p>
+  <%= render partial:'comments/index',
+locals: {answer: answer, comments: answer.comments} %>
+<%= render partial: 'votes/form',locals: {parent: Answer.find_by(id: @question.id), vote: Vote.new} %>
```

Comment:
Try to be consistent with indentation and line breaks. This bit is hard to read.